### PR TITLE
add wirerope and methodtools

### PR DESCRIPTION
**Pull request by timkpaine**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by timkpaine** [comment]
@conda-forge/help-python all set!

File: recipes/methodtools/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+about:
+  home: https://github.com/youknowone/methodtools
+  license: BSD-2-Clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: Expand standard functools to methods
+
+  description: |
+    Expand standard functools to methods
```
Comment (BastianZim) [inline recipes/methodtools/meta.yaml:37]: If it's a duplicate, you don't need to include it.

File: recipes/methodtools/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=2.7
+    - pip
+  run:
```
Comment (BastianZim) [inline recipes/methodtools/meta.yaml:21]: For python 2, functools32 is required: https://github.com/youknowone/methodtools#installation

As conda-forge doesn't support Python 2 anymore, I would change the bound to 3.6.

**Review by BastianZim — CHANGES_REQUESTED** [review]
Just two small things and this is good to go.

**Comment by BastianZim** [comment]
Looks good. Please ping me again when the CI is finished and I'll merge it.

**Review by BastianZim — APPROVED** [review]

**Pull request merged**